packages/cli/src/sso/saml/__tests__/saml-validator.test.ts

```diff
@@ -0,0 +1,347 @@
+import { validateMetadata, validateResponse } from '../saml-validator';
+
+describe('saml-validator', () => {
+	describe('validateMetadata', () => {
```

Comment:
To reduce noise in tests:

```suggestion
import { Logger } from '@/logging/logger.service';
import { mockInstance } from '@test/mocking';

import { validateMetadata, validateResponse } from '../saml-validator';

describe('saml-validator', () => {
	mockInstance(Logger);

	describe('validateMetadata', () => {
```